tests/suit/tests_suite_test.go

```diff
@@ -13,10 +13,14 @@ import (
 )
 
 const (
-	networkMapName        = "network-map-test"
-	test_migration_name   = "migration-test"
-	test_plan_name        = "plan-test"
-	test_storage_map_name = "test-storage-map-v"
+	networkMapName                 = "network-map-test"
```

Comment:
pretty strange that only `networkMapName` is in camelcase convention while the rest is snake case. but it's not in yours change.

Reply:
agreed:) I wanted to change that but I think it was used in some other places so I left it as is to not include irreverent changes in this PR.